sites/en/installfest/windows.step

```diff
@@ -53,13 +46,13 @@ step "Update Git" do
 
   message "Run the installer, and it will ask you where you want to install it. Change it FROM C:\\Program Files\\Git  TO, C:\\RailsInstaller\\Git like the picture below."
 
-  img src: 'img/directory.png'
+  img src: 'img/directory.png', "Screenshot showing the step of installation where the location to install git is set"
```

Comment:
Looks like you missed the `alt:` here, which is causing this page not to render / the tests to fail.